src/main/java/io/confluent/copycat/hdfs/Data.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.copycat.hdfs;
+
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Data {
```

Comment:
This is kind of a weird class -- it seems to just be a way to share some in-memory data. Is there anywhere else we could store this HashMap.

Also, it looks like this and a bunch of the in-memory classes are used only for testing. Should they be moved under `src/test/` instead of `src/main/`?